jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -188,7 +188,32 @@ public static DslThreadGroup threadGroup(String name) {
    * @since 0.33
    */
   public static DslSetupThreadGroup setupThreadGroup(ThreadGroupChild... children) {
-    return new DslSetupThreadGroup(Arrays.asList(children));
+    return new DslSetupThreadGroup(null, Arrays.asList(children));
+  }
+
+  /**
+   * This allowing to set a name on the setup thread group.
+   * <p>
+   * Setting a proper name allows to properly identify the requests generated in each thread group.
+   *
+   * @see DslSetupThreadGroup
+   * @since 0.33.2
```

Comment:
let's make it 0.34 since adds new methods and way of interacting (improvement). 

We are currently only increasing patch for bug fixes. This way users know what to expect: just fixes (patch version), new features/ways of interact (minor), non backwards compatible and big changes (since we are in 0, since core API is still not 100% stable, some minor changes still include non backwards compatible changes, but whenever we decide to do the first major release this should not happen any more). 

This is our interpretation of semver versioning applied to the the DSL :), I know it might not be the same interpretation as other projects.

We can do the change after merge anyways :-)